### PR TITLE
Migrate to TMS

### PR DESCRIPTION
**Pull request by bsberry**
* Change all TT references to TMS
* Update scripts to use latest tools and comment out errors in building.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,11 +37,11 @@ Download
 --------
 
 ```groovy
-compile 'com.twotoasters.servos:util:1.0.0'
-compile 'com.twotoasters.servos:util-butterknife:1.0.0'
-compile 'com.twotoasters.servos:util-otto:1.0.0'
-compile 'com.twotoasters.servos:util-otto-retrofit:1.0.0'
-compile 'com.twotoasters.servos:util-picasso:1.0.0'
+compile 'com.ticketmaster.servos:util:1.0.0'
```
Comment (patjackson52) [inline README.md]: Discussion:
There's been talk of using com.ticketmaster.mobilestudio as our package.  Sure we want com.ticketmaster?

Comment (bsberry) [inline README.md]: First I've heard of it. I just picked something and ran with it.

Comment (mgibble) [inline README.md]: the purchase SDK team has chosen com.ticketmaster.mobilestudio

**Comment by curtinmartis** [comment]
lgtm

**Comment by patjackson52** [comment]
+1

**Pull request merged**